sources/osg/FrameBufferObject.js

```diff
@@ -252,6 +252,10 @@ FrameBufferObject.prototype = MACROUTILS.objectInherit( GLObject.prototype, MACR
         if ( status !== 0x8CD5 ) {
             this._reportFrameBufferError( status );
         }
+        if ( gl.checkFramebufferStatus( gl.FRAMEBUFFER ) !== gl.FRAMEBUFFER_COMPLETE ) {
```

Comment:
you're adding exact same gl call as lines above no ?
the gl.checkFramebufferStatus then status !== 0x8Cd5 
cannot remember if do we call checkstatus at each call or only at creation ?